src/components/SoraCard/SoraCardKYC.vue

```diff
@@ -1,14 +1,13 @@
 <template>
-  <wallet-base v-loading="loading" :title="title" :show-back="showBack" @back="handleBack" class="sora-card">
+  <wallet-base v-loading="loading" :title="title" :show-back="true" @back="handleBack" class="sora-card">
     <terms-and-conditions v-if="step === KycProcess.TermsAndConditions" @confirm-tos="confirmToS" />
-    <road-map v-else-if="step === KycProcess.RoadMap" @confirm-start="confirmProcess" />
-    <kyc-view v-else-if="step === KycProcess.KycView" @confirm-kyc="confirmWindow" :accessToken="accessToken" />
-    <confirmation-info v-else-if="step === KycProcess.ConfirmationInfo" />
+    <road-map v-else-if="step === KycProcess.RoadMap" @confirm-start-kyc="confirmProcess" :userApplied="userApplied" />
+    <kyc-view v-else-if="step === KycProcess.KycView" @confirm-kyc="redirectToView" />
```

Comment:
could we use `<component :is="" />` here? all events might be like `confirm`...

Reply:
code will become overcomplicated, these components will not always bring forward/backwards sequentially throughout flow. Some can lead to other views completely, this results in passing even more variables and respectful if-clauses to check where to go next